src/components/Form/Form.js

```diff
@@ -58,21 +46,33 @@ export function Form({ children, config, render }) {
       await config.onSubmit(getInputValues())
       alert(
         'form submission successful with values:' +
-          JSON.stringify(getInputValues())
+          JSON.stringify(getInputValues()) +
+          "\n\nBut try again, it's random"
       )
     } catch (e) {
       setFormErrorMessage('Something bad happened: ' + e.toString())
     }
   }
 
+  function input(inputName) {
+    const inputConfig = config.inputs[inputName]
```

Comment:
might as well start using the vernacular fields for all of this no? Or is that v2b? :)

Reply:
But that's the direction you have in mind for later yah??